jest/private/jest_test.bzl

```diff
@@ -3,16 +3,21 @@
 load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
 load("@aspect_bazel_lib//lib:paths.bzl", "relative_file")
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
+load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _attrs = dicts.add(js_binary_lib.attrs, {
     "entry_point": attr.label(
         mandatory = True,
     ),
     "config": attr.label(
         doc = "jest config file, see TODO",
-        allow_single_file = [".js"],
+        allow_single_file = [".js", ".cjs", ".mjs", ".json", ".ts"],
```

Comment:
nit: link back to https://jestjs.io/docs/configuration as the source of this list